frontend/src/libs/components/modal/modal.tsx

```diff
@@ -0,0 +1,62 @@
+import { createRef } from 'react';
+
+import { Icon, Portal } from '~/libs/components/components.js';
+import { useCallback, useEffect } from '~/libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  children: React.ReactNode;
+  onClose: () => void;
+  isOpen: boolean;
+};
+
+const Modal: React.FC<Properties> = ({
+  isOpen,
+  onClose: handleOnClose,
```

Comment:
I guess there's no need to rename the prop

Reply:
Handlers located in the component must have the `handle` prefix.

Reply:
Hmm, but actually it's a prop, not a handler. Maybe I'm wrong 😅 @andanf-e could you please suggest here

Reply:
@artemmatiushenko1 you are right
Yep, it's a handler passed as a prop, so it s hould be named as it's passed -- actually may reason with such kind of naming convention is to differentiate own component handlers (`handleKek`) and passed methods (`onKek`)

Reply:
> Handlers located in the component must have the `handle` prefix.

This is a quote from Mykola when I used `onClose` as an `onClick` hanler. OK, I will change it again)

Reply:
Thanks